cristin-import/src/test/resources/features/GeneralMappingRules.feature

```diff
@@ -211,6 +211,19 @@ Feature: Mappings that hold for all types of Cristin Results
     When the Cristin Result is converted to an NVA Resource
     Then the NVA Resource has no projects
 
+  Scenario: The Cristin Result's HRCS values are used to generate the URIs for the NVA Resource
```

Comment:
Do we take care of helsekategorikode and aktivitetskode values that have the expected format but they do not exist?